styles.css

```diff
@@ -0,0 +1,140 @@
+body {
+  background-color: #e5e5e5;
+  margin: 0;
+  padding: 0;
+}
+
+.header-logo {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+
+  /* position: fixed; */
```

Comment:
Kindly remove all commented out code from your project to keep it clean🙏

Reply:
Done